packages/web/src/shared/components/Link.tsx

```diff
@@ -0,0 +1,10 @@
+import { Link as BaseLink, LinkProps } from 'react-router-dom';
+
+export function Link({ className, ...props }: LinkProps) {
+  return (
+    <BaseLink
```

Comment:
All of these generic shared components should help us keep a consistent style as we build new views